### ISSUE TITLE
laminas/laminas-mail has been abandoned

### ISSUE THREAD

**Issue by marclaporte** [Webador/SlmMail#162]
https://github.com/laminas/laminas-mail/blob/2.26.x/README.md
https://github.com/laminas/technical-steering-committee/blob/main/meetings/minutes/2023-12-04-TSC-Minutes.md

**Comment by roelvanduijnhoven** [comment]
Mm that sucks @marclaporte. Thanks for bringing that my attention. Weird that there is no rationale to be found about the move. 

This package builds quite heavily on top of that package. So I don't think this repo will / should diverge onto something else. 

I guess we can just keep this package around for now. And keep depending on the abandon ware. Although maybe we _should_ make note of that on the repo page?

**Comment by marclaporte** [comment]
> Weird that there is no rationale to be found about the move.

@weierophinney is the Matthew referenced here:
"Matthew indicated that he feels without anybody who can commit to maintaining it, and nobody on the team with security experience in this particular area, it may actually be a disservice to end users to commit to security patches."
Source: https://github.com/laminas/technical-steering-committee/blob/main/meetings/minutes/2023-12-04-TSC-Minutes.md

**Comment by marclaporte** [comment]
For analysis:

Other users of laminas-mail:
https://packagist.org/packages/laminas/laminas-mail/dependents?order_by=downloads

Other users of slm/mail:
https://packagist.org/packages/slm/mail/dependents?order_by=downloads

Also, the Tiki Wiki CMS Groupware project uses:
https://gitlab.com/tikiwiki/tiki/-/blob/24.x/vendor_bundled/composer.json#L139
https://doc.tiki.org/Transactional-Email-Providers

**Comment by marclaporte** [comment]
> Although maybe we _should_ make note of that on the repo page?

I agree. And perhaps suggest alternatives. Unless some steps up for laminas-mail, no one should start new projects which depend on it (direclty or indirectly)

**Comment by roelvanduijnhoven** [comment]
@marclaporte I can put up a warning at the top of the README about the state of laminas/mail. No problem.

But I'm not aware what alternatives to suggest. Do you hve some recommendations I can put in there?

**Comment by marclaporte** [comment]
@roelvanduijnhoven fyi above. Tks.

**Comment by roelvanduijnhoven** [comment]
Thanks. I've updated the README on master @marclaporte!
